src/apiCalls.tsx

```diff
@@ -46,13 +47,21 @@ const checkResponse = (response: Response) => {
   return response.json()
 }
 
+export const getSentiment = (url: string) => {
+  return fetch(`https://api.dandelion.eu/datatxt/sent/v1/?lang=en&url=${url}&token=295d5b1d94bb4af5a2d229cf261a449e`)
+    .then(res => res.json())
```

Comment:
Could we add the checkResponse(response) to ensure all errors are caught?

Reply:
Added `checkResponse`, thank you!